src/components/SelectionList/BaseSelectionList.tsx

```diff
@@ -64,20 +64,23 @@ function BaseSelectionList<TItem extends ListItem>(
         customListHeader,
         listHeaderWrapperStyle,
         isRowMultilineSupported = false,
+        textInputRef,
     }: BaseSelectionListProps<TItem>,
-    inputRef: ForwardedRef<RNTextInput>,
+    ref: ForwardedRef<SelectionListHandle>,
 ) {
     const styles = useThemeStyles();
     const {translate} = useLocalize();
     const listRef = useRef<RNSectionList<TItem, Section<TItem>>>(null);
-    const textInputRef = useRef<RNTextInput | null>(null);
+    const innerTextInputRef = useRef<RNTextInput | null>(null);
     const focusTimeoutRef = useRef<NodeJS.Timeout | null>(null);
     const shouldShowTextInput = !!textInputLabel;
     const shouldShowSelectAll = !!onSelectAll;
     const activeElementRole = useActiveElementRole();
     const isFocused = useIsFocused();
     const [maxToRenderPerBatch, setMaxToRenderPerBatch] = useState(shouldUseDynamicMaxToRenderPerBatch ? 0 : CONST.MAX_TO_RENDER_PER_BATCH.DEFAULT);
     const [isInitialSectionListRender, setIsInitialSectionListRender] = useState(true);
+    const [itemToHighlight, setItemToHighlight] = useState<Set<string> | null>(null);
```

Comment:
This can be a list of items, right?
```suggestion
    const [itemsToHighlight, setItemsToHighlight] = useState<Set<string> | null>(null);
```